src/Utility/Xml.php

```diff
@@ -328,6 +328,7 @@ protected static function _fromArray($dom, $node, &$data, $format)
      */
     protected static function _createChild($data)
     {
+        $value = $dom = $key = $format = $node = null;
```

Comment:
I would just get rid of the remaining extract()s in one of the next PRs/commits.